pagopa/pn-frontend-e2e
# PN-9323-e-2-e-mittente-inserimento-data-errata

## Short description
There isn't a test related to the **PN-9323** as you can see in the [confluence page](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868778282/Mittente+-+Mapping+test-scenari)

## List of changes proposed in this pull request
- added new test file `010_1_ricercaPerCodiceDataErrata.feature`
- added new feature steps such as:
```
And Si verifica che i campi della ricerca delle date siano errate
And Si verifica che il bottone Filtra sia disabilitato
```
- added related checks and methods


## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_MittenteRicercaPerDataErrata` and verify if the test works fine.
